sdk/network/Azure.ResourceManager.Network/src/Generated/Models/ConnectionMonitorEndpoint.cs

```diff
@@ -86,15 +90,31 @@ internal ConnectionMonitorEndpoint()
         public string Name { get; set; }
         /// <summary> The endpoint type. </summary>
         public ConnectionMonitorEndpointType? EndpointType { get; set; }
-        /// <summary> Resource ID of the connection monitor endpoint. </summary>
+        /// <summary> Resource ID of the connection monitor endpoint are supported for AzureVM, AzureVMSS, AzureVNet, AzureSubnet, MMAWorkspaceMachine, MMAWorkspaceNetwork, AzureArcVM endpoint type. </summary>
         public ResourceIdentifier ResourceId { get; set; }
-        /// <summary> Address of the connection monitor endpoint (IP or domain name). </summary>
+        /// <summary> Address of the connection monitor endpoint. Supported for AzureVM, ExternalAddress, ArcMachine, MMAWorkspaceMachine endpoint type. </summary>
         public string Address { get; set; }
-        /// <summary> Filter for sub-items within the endpoint. </summary>
+        /// <summary> Filter field is getting deprecated and should not be used. Instead use Include/Exclude scope fields for it. </summary>
         public ConnectionMonitorEndpointFilter Filter { get; set; }
-        /// <summary> Endpoint scope. </summary>
+        /// <summary> Endpoint scope defines which target resource to monitor in case of compound resource endpoints like VMSS, AzureSubnet, AzureVNet, MMAWorkspaceNetwork, AzureArcNetwork. </summary>
         public ConnectionMonitorEndpointScope Scope { get; set; }
         /// <summary> Test coverage for the endpoint. </summary>
         public CoverageLevel? CoverageLevel { get; set; }
+        /// <summary> Location details is optional and only being used for 'AzureArcNetwork' type endpoints, which contains region details. </summary>
+        internal ConnectionMonitorEndpointLocationDetails LocationDetails { get; set; }
+        /// <summary> Region for connection monitor endpoint. </summary>
+        public string LocationDetailsRegion
+        {
+            get => LocationDetails is null ? default : LocationDetails.Region;
+            set
+            {
+                if (LocationDetails is null)
+                    LocationDetails = new ConnectionMonitorEndpointLocationDetails();
+                LocationDetails.Region = value;
+            }
+        }
+
+        /// <summary> Subscription ID for connection monitor endpoint. It's an optional parameter which is being used for 'AzureArcNetwork' type endpoint. </summary>
+        public string SubscriptionId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).

Reply:
megha.jain Please check this comment

Reply:
Hi @archerzz ,
It's kind of guid and it's an optional parameter.

Reply:
Added